packages/paste-design-tokens/tokens/themes/twilio-dark/global/background-color.yml

```
@@ -11,6 +11,8 @@ props:
   color-background-inverse-strong:
     value: "{!palette-gray-90}"
     comment: Strong inverse background color for any container. Must be used on color-background-body-inverse.
+  color-background-weaker:
+    value: "{!palette-gray-0}"
```

Based on the table in the doc shouldn't this be 100?

```suggestion
    value: "{!palette-gray-100}"
```